src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -190,19 +190,23 @@ def loadAllPerspectives(self):
         # Load all perspectives
         loaded_dict = {}
         for name, perspective in Perspectives.PERSPECTIVES.items():
-            loaded_perspective = perspective(parent=self)
-            loaded_dict[name] = loaded_perspective
+            try:
+                loaded_perspective = perspective(parent=self)
+                loaded_dict[name] = loaded_perspective
+            except Exception as e:
+                logger.log(f"Unable to loading {name} perpesctive.\n{e}")
```

Comment:
`f"Unable to load {name} perspective.\n{e}"`

Reply:
I no talk English good...